modules/FormView.js

```diff
@@ -382,6 +387,29 @@
     },
 
     /**
+     * @method  _generateSelectOptions
+     * @param element {Element} The select element to generate options for
+     * @param opts {Object} Additional behavior options for the bindings
+     * @param [options.modelFormat] {Object} The function called before setting model values
+     * @private
+     * @return {<Stickit select options hash>}
+     */
+    _generateSelectOptions: function(element, opts) {
+      var collection = [],
+          options = $(element).children('option');
+
+      _.each(options, function(option) {
+        collection.push({'label': $(option).text(),
+                          'value': opts.modelFormat ? opts.modelFormat.apply(this, $(option).val()) : $(option).val()});
+      });
+
+      return {collection: collection,
```

Comment:
Add an issue to Torso that explains the bug and reference the issue in this commit message.